corehq/apps/app_manager/templates/app_manager/releases.html

```diff
@@ -210,6 +210,7 @@ <h6 data-bind="if: !built_with.signed()">{% trans "Unsigned Jar" %}</h6>
                                     generate_short_url('short_odk_url');
                                     ga_track_event('App Manager', 'Deploy Button', '{{ app.id }}');
                                     analytics.workflow('Clicked Deploy');
+                                    $.post('/analytics/hubspot/click-deploy/')
```

Comment:
if you add the `name` param, you can do a `reverse` here instead of hardcoding the url